exporter/stackdriverexporter/config.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package stackdriverexporter
+
+import "github.com/open-telemetry/opentelemetry-service/config/configmodels"
+
+// Config defines configuration for Stackdriver exporter.
+type Config struct {
+	configmodels.ExporterSettings `mapstructure:",squash"` // squash ensures fields are correctly decoded in embedded struct.
+	ProjectID                     string                   `mapstructure:"project"`
+	EnableTracing                 bool                     `mapstructure:"enable_tracing"`
```

Comment:
We have to make either `_` or `-` the standard for composed entries in the config. My personal preference is for dash, but I'm fine with underscore if preferred by the majority.